test/gie/builtins.gie

```diff
@@ -2021,6 +2021,29 @@ expect  -0.001790493 0.000759909
 accept  -200 -100
 expect  -0.001790493 -0.000759909
 
+-------------------------------------------------------------------------------
+operation +proj=healpix   +R=6400000   +lat_1=0.5 +lat_2=2  +rot_xy=42
+-------------------------------------------------------------------------------
+tolerance 0.1 mm
+accept  2 1
```

Comment:
to test just that parameter, one expect/accept tuple should be enough.
But I find it more convincing to use the expect value as the acept value of the `direction inverse` case. This shows correct implementation of the roundtripping